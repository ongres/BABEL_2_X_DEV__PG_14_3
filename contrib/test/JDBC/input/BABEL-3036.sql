CREATE LOGIN tester WITH PASSWORD = '12121212';
GO

CREATE USER test_user FOR LOGIN tester;
GO

SELECT suser_sname(sid) from database_principals where name = 'test_user';
GO

DROP USER test_user;
GO

DROP LOGIN tester;
GO
