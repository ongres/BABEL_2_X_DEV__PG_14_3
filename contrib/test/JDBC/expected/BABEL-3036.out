CREATE LOGIN tester WITH PASSWORD = '12121212';
GO

CREATE USER test_user FOR LOGIN tester;
GO

SELECT suser_sname(sid) from database_principals where name = 'test_user';
GO
~~START~~
nvarchar
tester
~~END~~


DROP USER test_user;
GO

DROP LOGIN tester;
GO
